packages/cells/src/model.ts

```diff
@@ -424,7 +424,9 @@ export class RawCellModel extends AttachmentsCellModel {
    * Serialize the model to JSON.
    */
   toJSON(): nbformat.IRawCell {
-    return super.toJSON() as nbformat.IRawCell;
+    const cell = super.toJSON() as nbformat.IRawCell;
+    cell.id = this.id;
```

Comment:
Can we do this in the super class's method, the one we are calling with `super.toJSON()`?

Reply:
The reason I had done it this way was because the notebook 4.5 specification only has the `id` field on raw cells, code cells, and markdown cells. It's not part of the specification for urecognized cells. 

https://github.com/jupyter/nbformat/blob/master/nbformat/v4/nbformat.v4.5.schema.json